core/src/scene/sceneLoader.cpp

```diff
@@ -101,7 +101,15 @@ bool SceneLoader::loadScene(Node& config, Scene& _scene) {
     std::sort(_scene.styles().begin(), _scene.styles().end(),
               [](std::unique_ptr<Style>& a, std::unique_ptr<Style>& b) {
                   if (a->blendMode() != b->blendMode()) {
-                      return a->blendMode() == Blending::none;
+                      if (a->blendMode() == Blending::none || b->blendMode() == Blending::none) {
+                          return a->blendMode() == Blending::none;
```

Comment:
Refer to newer commits. I think your suggestion  definitely makes the code more concise.